src/options/set.rs

```diff
@@ -656,4 +660,23 @@ pub mod tests {
 
         remove_file(git_config_path).unwrap();
     }
+
+    #[test]
+    fn test_width_in_git_config_is_honored() {
+        let git_config_contents = b"
+[delta]
+    width = 7
```

Comment:
I think we should make this test value "variable". Otherwise there is a possibility that the test will be running in a terminal which really does have the width value 7!

Reply:
thanks to your comment https://github.com/dandavison/delta/pull/265#issuecomment-665825388 as well, fixed here
00f3899